### PR TITLE
Merge 1.7.5.x into develop

### PR DESCRIPTION
**Pull request by PierreRambaud**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Get latest changes from 1.7.5.x
| Type?         | improvement
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| How to test?  | Wait for Travis ci and :pray:

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/11072)

### CODE REVIEW

**Review by matks — DISMISSED** [review]

**Review by jolelievre — DISMISSED** [review]
nice!

File: admin-dev/themes/default/js/bundle/product/catalog.js [hunk 1]
```diff
@@ -371,70 +372,70 @@ function unitProductAction(element, action) {
...
-
-			$('#bulk_edition_toolbar input:submit').removeAttr('editionaction');
-			showBulkProductEdition(false);
-			$('input#bulk_action_select_all, input:checkbox[name="bulk_action_selected_products[]"]', form).prop('disabled', false);
-			break;
-	}
+  var form = $('form#product_catalog_list');
+
+  switch (action) {
+    /*
```
Comment (mickaelandrieu) [inline admin-dev/themes/default/js/bundle/product/catalog.js:391]: is commented code a right thing here?

Comment (Quetzacoalt91) [reply admin-dev/themes/default/js/bundle/product/catalog.js:391]: Well, it's as on the `1.7.5.x` branch: https://github.com/PrestaShop/PrestaShop/blob/7a079d7ade205ea5b9380b5f4c4ab04b848f989b/admin-dev/themes/default/js/bundle/product/catalog.js#L392

Comment (matks) [reply admin-dev/themes/default/js/bundle/product/catalog.js:391]: It's commented too on `develop`
https://github.com/PrestaShop/PrestaShop/blob/develop/admin-dev/themes/default/js/bundle/product/catalog.js#L391
This diff is strange 🤔 and I'm not confident with a strange diff for such a critical PR ...

Comment (Quetzacoalt91) [reply admin-dev/themes/default/js/bundle/product/catalog.js:391]: No need to worry about, you can see on the diff that the indentation of the whole file has been updated, that's why you get changes.
And we're not even talking about conflicts on this case.

**Comment by Quetzacoalt91** [comment]
Just ran the E2E tests locally, and found the following issue:

capture du 2018-10-25 14-57-08

**Comment by matks** [comment]
🔥 we need to merge this ^^

**Comment by Quetzacoalt91** [comment]
I know, this is still in progress!

```
  98 passing (1m)

```

**Review by matks — DISMISSED** [review]

**Comment by Quetzacoalt91** [comment]
Updated PR to make an actual merge instead of rebase.

File: src/PrestaShopBundle/Resources/config/services/bundle/form/form_handler.yml [hunk 2]
```diff
@@ -148,7 +148,7 @@ services:
             - 'LogsPage'
 
     prestashop.admin.import.form_handler:
-        class: 'PrestaShop\PrestaShop\Core\Form\FormHandler'
+        class: 'PrestaShop\PrestaShop\Core\Form\ImportPageFormHandler'
         arguments:
             - '@prestashop.admin.import.form_builder'
```
Comment (mickaelandrieu) [inline src/PrestaShopBundle/Resources/config/services/bundle/form/form_handler.yml:153]: why do we have a specific form builder for import?

**Comment by Quetzacoalt91** [comment]
To fix this issue:

capture du 2018-10-29 16-38-45

File: src/PrestaShopBundle/Resources/config/services/bundle/form/form_handler.yml (hunk 2 above)
Comment (Quetzacoalt91) [reply src/PrestaShopBundle/Resources/config/services/bundle/form/form_handler.yml:153]: To fix this issue:

capture du 2018-10-29 16-38-45

**Review by matks — APPROVED** [review]

**Pull request merged**